pipes/rules/metagenomics.rules

```diff
@@ -29,16 +30,29 @@ rule kraken:
     resources: cores=int(config.get("number_of_threads", 1))
     params: numThreads=str(config.get("number_of_threads", 1))
     run:
+        shell("pyenv which python")
```

Comment:
are these debugging lines? I'd propose removing them, unless we want to introduce them to more of the rules. But actually, I think util.cmd starts off each new execution of a script with the python version. We could probably have it also report the path-to-python as well since that would be nice debugging info.